modular_skyrat/modules/cyborg/code/mechafabricator_designs.dm

```diff
@@ -52,3 +52,12 @@
 	)
 	category = list(RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_ENGINEERING)
 
+/datum/design/borg_shapeshifter_module
```

Comment:
![image](https://user-images.githubusercontent.com/82386923/212967202-1b04a7e0-5e74-4e30-b89e-a040b7723ce1.png)

Reply:
? I took it out of its own file and brought it into our mechfab modular file